### PR TITLE
feat: update scalar deps

### PR DESCRIPTION
**Pull request by emmerich**
Reopening #2303

### CODE REVIEW

File: packages/react-openapi/src/ScalarApiButton.tsx [hunk 1]
```diff
@@ -98,7 +98,10 @@ export function ScalarApiClient(props: { children: React.ReactNode }) {
             {
                 url: operationData.path,
             },
-            getRequestFromOperation(operation, { requiredOnly: false }),
+            getRequestFromOperation(
+                { ...operation, information: operationData.operation },
```
Comment (emmerich) [inline packages/react-openapi/src/ScalarApiButton.tsx:102]: @marclave There might be some incompleteness on typing here: 

Screenshot 2024-05-01 at 15 25 54

Comment (marclave) [reply packages/react-openapi/src/ScalarApiButton.tsx:102]: ah! let me fix this : )

Comment (marclave) [reply packages/react-openapi/src/ScalarApiButton.tsx:102]: https://github.com/GitbookIO/gitbook/pull/2310/files

fixed :)

**Comment by emmerich** [comment]
Closing in favour of #2310

**Pull request closed without merging**